domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -37,4 +39,14 @@ public async Task<ActionResult> Create([FromBody] CreateTransferAgreement reques
 
         return Created($"api/transfer-agreements/{result.Id}", result);
```

Comment:
This has the location of the created resource and the route is hardcoded because, the GET Endpoint was not ready. It is ready now in this PR, so we can get rid of the the hardcoded route. Please consider using CreatedAtAction or CreatedAtRoute here, see this for more info https://ochzhen.com/blog/created-createdataction-createdatroute-methods-explained-aspnet-core